internal/provider/nutanix/installConfig.go

```diff
@@ -0,0 +1,52 @@
+package nutanix
+
+import (
+	"errors"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/internal/installcfg"
+)
+
+func setPlatformValues(platform *installcfg.NutanixInstallConfigPlatform) {
+	// Add placeholders to make it easier to replace in day2
+	platform.PrismCentral = installcfg.NutanixPrismCentral{
+		Endpoint: installcfg.NutanixEndpoint{
+			Address: PhPCAddress,
+			Port:    PhPCPort,
+		},
+		Username: PhUsername,
+		Password: PhPassword,
+	}
+	platform.PrismElements = []installcfg.NutanixPrismElement{
+		{
+			Endpoint: installcfg.NutanixEndpoint{
+				Address: PhPEAddress,
+				Port:    PhPEPort,
+			},
+			UUID: PhPUUID,
+		},
+	}
+	platform.SubnetUUIDs = []strfmt.UUID{
+		strfmt.UUID(PhSubnetUUID),
+	}
+}
+
+func (p nutanixProvider) AddPlatformToInstallConfig(
+	cfg *installcfg.InstallerConfigBaremetal, cluster *common.Cluster) error {
+	if len(cluster.APIVip) == 0 {
```

Comment:
think about UMN as well here

Reply:
Do we wan't to return error when UMN is enabled?

Reply:
If no we might want to to something similar to vsphere:

```go
	if !swag.BoolValue(cluster.UserManagedNetworking) {
		if len(cluster.APIVip) == 0 {
			return errors.New("invalid cluster parameters, APIVip must be provided")
		}

		if len(cluster.IngressVip) == 0 {
			return errors.New("invalid cluster parameters, IngressVip must be provided")
		}

		vsPlatform.APIVIP = cluster.APIVip
		vsPlatform.IngressVIP = cluster.IngressVip
	} else {
		cfg.Networking.MachineNetwork = provider.GetMachineNetworkForUserManagedNetworking(p.Log, cluster)
		if cluster.NetworkType != nil {
			cfg.Networking.NetworkType = swag.StringValue(cluster.NetworkType)
		}
	}


```

Reply:
Lets make it similar to vSphere and run another round of tests. If UMN is confirmed to be impossible on Nutanix we'll disable it entirely

Reply:
for some reason, i recalled that @rvanderp3 said it should work, just not tested. But i might be wrong here